transmartApp/grails-app/views/ontology/_showDefinition.gsp

```diff
@@ -13,7 +14,20 @@
     <g:if test="${tags}">
         <g:render template="showTags" model="tags"/>
     </g:if>
-    <g:if test="${browseStudyInfo}">
-        <g:render template="showBrowseStudyInfo" model="browseStudyInfo"/>
+<%--    <g:if test="${browseStudyInfo}">--%>
```

Comment:
are those commented codes still used? If not pls remove, if yes pls add TODO about what is it for.